src/main/java/com/google/ehub/data/EntertainmentItem.java

```diff
@@ -4,16 +4,22 @@
  * Holds information for Entertainment Items used in Datastore.
  */
 public final class EntertainmentItem {
+  private final long uniqueID;
   private final String title;
   private final String description;
   private final String imageURL;
 
-  public EntertainmentItem(String title, String description, String imageURL) {
+  public EntertainmentItem(long uniqueID, String title, String description, String imageURL) {
```

Comment:
nit: `uniqueId`

Reply:
I renamed "ID" to "Id" across Item classes in last commit.